docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts

```diff
@@ -55,7 +81,15 @@ export class CustomI18nLabels extends CalendarI18nLabels {
 @Component({
     selector: 'fd-datepicker-i18n-example',
     template: `
-        <fd-date-picker [(ngModel)]="date"></fd-date-picker>`,
+        <label fd-form-label for="language">Select language:</label>
+        <fd-button-group id="language" style="margin-bottom:20px">
+            <button fd-button-grouped [size]="'xs'" (click)="setFrench()" [state]="selected === 1 ? 'selected' : ''">French</button>
```

Comment:
you should use get methods instead of boolean expressions in the template where possible. Doing so makes code self-documenting and easier to read/follow.